cli/wallet/wallet.go

```diff
@@ -54,10 +59,16 @@ func NewCommands() []cli.Command {
 				Action: dumpWallet,
 				Flags: []cli.Flag{
 					walletPathFlag,
-					cli.BoolFlag{
-						Name:  "decrypt, d",
-						Usage: "Decrypt encrypted keys.",
-					},
+					decryptFlag,
+				},
+			},
+			{
+				Name:   "export",
+				Usage:  "export keys for address",
```

Comment:
Again, positional arguments need additional comments right in the help section. Users are gonna be invoking `help` to find out how to use this thing and it should tell them how.

Reply:
Added `UsageText` which is shown when help for a specific command is requested.